<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>王颜婷2020060903026</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="textLabel1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>原图像</string>
    </property>
   </widget>
   <widget class="QListView" name="HistoricalOprations_ListView">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>241</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="labelCapture">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="processing_Image">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>处理后图像</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>450</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具(T)</string>
    </property>
    <widget class="QMenu" name="noise">
     <property name="title">
      <string>噪声</string>
     </property>
     <addaction name="actionGauss_noise"/>
     <addaction name="actionPepper_noise"/>
    </widget>
    <widget class="QMenu" name="Color">
     <property name="title">
      <string>颜色处理</string>
     </property>
     <addaction name="action2Gray"/>
    </widget>
    <widget class="QMenu" name="menuEnhancement">
     <property name="title">
      <string>增强</string>
     </property>
     <addaction name="actionHist"/>
     <addaction name="actionLog"/>
    </widget>
    <addaction name="actionRecover"/>
    <addaction name="noise"/>
    <addaction name="menuEnhancement"/>
    <addaction name="Color"/>
    <addaction name="actionBlur"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开(O)</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(S)</string>
   </property>
  </action>
  <action name="actionGauss_noise">
   <property name="text">
    <string>高斯噪声</string>
   </property>
  </action>
  <action name="actionPepper_noise">
   <property name="text">
    <string>椒盐噪声</string>
   </property>
  </action>
  <action name="actionbox_filter">
   <property name="text">
    <string>盒式滤波</string>
   </property>
  </action>
  <action name="actionclearMenu">
   <property name="text">
    <string>清除</string>
   </property>
  </action>
  <action name="actionRecover">
   <property name="text">
    <string>恢复</string>
   </property>
  </action>
  <action name="action2Gray">
   <property name="text">
    <string>灰度化</string>
   </property>
  </action>
  <action name="action2RGB">
   <property name="text">
    <string>彩色化</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>模糊</string>
   </property>
  </action>
  <action name="actionHist">
   <property name="text">
    <string>直方图增强</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>对数增强</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
